torchcde/interpolation_linear.py

```diff
@@ -91,6 +142,9 @@ def linear_interpolation_coeffs(x, t=None):
             length-many observations. Missing values are supported, and should be represented as NaNs.
         t: Optional one dimensional tensor of times. Must be monotonically increasing. If not passed will default to
             tensor([0., 1., ..., length - 1]).
+        rectilinear: Optional integer. Used for performing rectilinear interpolation. Default is None which results in
+            standard linear interpolation. For rectilinear interpolation time *must* be a channel in x and the
+            `rectilinear` parameter must be an integer specifying the channel index location of the time index in x.
```

Comment:
It would be nice to have a (brief one sentence) description of what rectilinear interpolation is here.